pubsublite/consumer_test.go

```diff
@@ -37,37 +37,3 @@ func TestNewConsumer(t *testing.T) {
 		assert.ErrorContains(t, err, "pubsublite: delivery is not valid")
 	})
 }
-
-func TestSubscriptionString(t *testing.T) {
```

Comment:
Should there be tests for `Validate()`?
(maybe in another PR)

Reply:
Yes probably, in a followup.

Reply:
I expanded the tests and validation logic in this PR, added a commit.